lib/tizen_plugins.dart

```diff
@@ -469,27 +469,24 @@ using System;
 using System.Runtime.InteropServices;
 using Tizen.Flutter.Embedding;
 
-namespace Runner
+internal class GeneratedPluginRegistrant
```

Comment:
@WonyoungChoi The `Runner` namespace has been removed from `GeneratedPluginRegistrant` to avoid

```csharp
Runner::GeneratedPluginRegistrant.RegisterPlugins(flutterView.Engine);
```

when importing a module from an app that uses a different namespace. Is it okay to use the global namespace?

Reply:
Yes, I think it doesn't matter.